### ISSUE TITLE
Is this library still under active development?

### ISSUE THREAD

**Issue by yeehaa123** [FacultyCreative/ngActiveResource#69]
I love this project, but there has not been any changes since september. Starting a new, long-term project and don't want to use ActiveResource if it is no longer maintained...

**Comment by AlJohri** [comment]
I found this to be a good alternative: https://github.com/jmdobry/angular-data
Another popular one is: https://github.com/mgonto/restangular

**Comment by timkendall** [comment]
I was in the same boat a couple of weeks ago - love this project but it's really not ready for production. I went with [https://github.com/jmdobry/angular-data](https://github.com/jmdobry/angular-data)

**Comment by brayniverse** [comment]
Angular-data definitely gets more love than this library and the documentation seems more thorough and thought out. But if you love this project @yeehaa123 maybe throw some PRs about and see if you can revive it a little?

**Comment by reneras** [comment]
@brayniverse +1

**Comment by demisx** [comment]
+1 would love to see this revived. Bummer, if it's abandoned.

**Comment by brettshollenberger** [comment]
Hi friends,

I would love to see folks step in and maintain the project (the most up-to-date code is over here now: https://github.com/brettshollenberger/the-abstractions-are-leaking). My work is now almost exclusively server-side, and I rarely have time or need to work on the project. If nothing else, I hope the project and the corresponding videos serve as good lessons in the concepts of data-modeling. 

Best,
Brett

**Comment by demisx** [comment]
@brettshollenberger I can help as much as I can, but I won't be able to pull this alone. The code is quite advanced and will take time to master, but I love the concept. Is it production ready? If it is, should we use the (FacultyCreative/ngActiveResource)[https://github.com/FacultyCreative/ngActiveResource] repo or the new [brettshollenberger/the-abstractions-are-leaking](https://github.com/brettshollenberger/the-abstractions-are-leaking) one? We have an app that needs to go to prod in a couple of months and we are desperately looking for a data model library, hopefully similar to ActiveRecord capabilities. Please advise.

**Comment by ghost** [comment]
Neither of these two is production ready. As the man said, he had no time nor need to maintain this. The second repo he provided is almost a full rewrite of this one. If you need something production ready, take a look at this https://github.com/jmdobry/angular-data (as mentioned in previous comments). It's being pretty heavily maintained (last time i checked).

**Comment by demisx** [comment]
Agreed. The new [angular-data 2.0](http://www.js-data.io/v1.3.0/docs/js-data-angular) (js-data + js-data-angular) works beautifully for client side ORM layer. And JSData support is phenomenal. I think this issue can be closed.

**Comment by gaboesquivel** [comment]
+1 js-data-angular
